client/sagas/frame-epic.js

```diff
@@ -47,6 +51,24 @@ function getFrameDocument(document, id = mainId) {
     frame = createFrame(document, id);
   }
   frame.contentWindow.loopProtect = loopProtect;
+
+  /* NOTE: provisioning depenendencies:
+   * Bind React & Redux depenendencies to content window so they are
+   * in scope during code evaluation and testing. These imports will
+   * add a lot of code to the app bundle... however, I believe that
+   * they can be included via the script tag header above to simply
+   * load them dynamically from CDNs and into this frame when it
+   * loads on the client */
+  frame.contentWindow.React = React;
```

Comment:
We don't want to bind react challenges to the react used by the app. This couples our app to the challenges and means we won't be able to upgrade the app without first considering whether all the challenges are not effected.

We have a mechanism to import required libraries for challenges built into the rechallenge framework. This should be where we import libraries for all challenges.